Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityController.cs

```diff
@@ -418,9 +418,24 @@ private void UpdateTriggerData(InteractionSourceState interactionSourceState, Mi
                             if (interactionSourceState.selectPressed)
                             {
                                 InputSystem?.RaiseOnInputDown(InputSource, ControllerHandedness, interactionMapping.MixedRealityInputAction);
+
+                                for (int i = 0; i < InputSource.Pointers.Length; i++)
+                                {
+                                    InputSystem?.RaisePointerDown(InputSource.Pointers[i], ControllerHandedness, interactionMapping.MixedRealityInputAction);
+                                }
                             }
                             else
                             {
+                                for (int i = 0; i < InputSource.Pointers.Length; i++)
+                                {
+                                    InputSystem?.RaisePointerClicked(InputSource.Pointers[i], ControllerHandedness, interactionMapping.MixedRealityInputAction, 1);
```

Comment:
The more I think about this change, the less certain I am. It changes the way pointer events work.

At the moment, pointers can be assigned an action to fire their pointer events. This becomes useful for, for example, a sphere-based grab pointer using the grip action. Now, only the trigger can fire those events, as we're explicitly firing them based on the `selectPressed` state.

At the very least, this change is unrelated to gestures, and I'd prefer not to block gestures on it.

Reply:
>**Removed Pointer Up/Down from all scripts raising events outside their respective device manager/controller classes.**